bin/cli.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+
+set -o noclobber
+set -o errexit
+set -o pipefail
+set -o nounset
+
+FONTS_DIRECTORY="fonts"
+RELEASE_DATE=$(date +'%B %d, %Y')
+STYLE="owncloud"
+STYLES_DIRECTORY="resources/themes"
+
+ERR_UNSUPPORTED_LANGUAGE=20
+ERR_UNSUPPORTED_MANUAL=21
+ERR_UNSUPPORTED_ACTION=22
+
+if [[ -z $VERSION ]]; then
+    if [[ -n $DRONE_TAG ]]; then
+        VERSION=${DRONE_TAG/v//}
+    else
+        [[ -n $DRONE_BRANCH ]] && VERSION=${DRONE_BRANCH/v//} || VERSION=master
+    fi
+fi
+
+function usage()
+{
+    echo "Usage: cmd [-c] [-b <admin|developer|user>] [-v <go|xml|json|php|kotlin|yaml>] -r <revision> -d <release date>"
+}
+
+function clean_build_dir()
+{
+    echo "Cleaning build directory..."
+    rm -rvf "build/"
+    echo "...build directory cleaned."
+}
+
+function validate_code_files()
+{
+    language="$1"
```

Comment:
Variables within functions should be defined as `local`.

Reply:
That's what I was forgetting. Thanks.